packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/lambdaLayerWalkthrough.ts

```diff
@@ -51,31 +144,34 @@ function layerNameQuestion(context: any) {
   ];
 }
 
-function layerPermissionsQuestion() {
+function layerPermissionsQuestion(params: Partial<LayerParameters>) {
```

Comment:
if this function only needs the layerPermissions, that should be the only input

Reply:
This assigns permissions in the layer parameters in this function. Thats why Params is passed

Reply:
This assigns permissions in the layer parameters in this function. Thats why Params is passed

Reply:
I'm saying it looks like this function only uses params.layerPermissions so why not just pass that in directly?